tests/test_io/good/all_io.catala_en

```diff
@@ -19,18 +19,31 @@ scope A:
 
 ```catala-test-inline
 $ catala Dcalc -s A
-let scope A (A_in: A_in): A =
-  let get c : integer = A_in."c_in" in
-  let get d : integer = A_in."d_in" in
-  let get e : unit → integer = A_in."e_in" in
-  let get f : unit → integer = A_in."f_in" in
+let scope A (A_in: A_in
+                     {c_in:
+                     integer;
+                     d_in:
+                     integer;
+                     e_in:
+                     unit → integer;
+                     f_in:
+                     unit → integer}): A
+                                           {b:
```

Comment:
This is still weird!

Reply:
Indeed! I focused on the expression printer for now, but the types printer also needs more love!

Reply:
Actually that was the scope-level printer that was added recently that had some issue; it should now be improved as well.